test/XtraBBroRewarder.spec.ts

```diff
@@ -55,7 +56,9 @@ describe("BBroPremiumStakingRewards", function () {
   })
 
   it("should allow whitelisted accounts to claim with extra perc", async function () {
```

Comment:
This is for Terra migrators right?
In case there's no test case for terra migrators yet, please add one.